modules/libcom/src/osi/osiSock.h

```diff
@@ -28,185 +34,367 @@ struct sockaddr;
 };
+
+/*!
+ * \brief Query what approach to use to interrupt blocking socket calls
+ *
+ * This function tells you what approach to use to interrupt calls blocked on
+ * socket operations.
+ *
+ * \return enum representing whether to call close() or shutdown(RW) to
+ * interrupt blocking socket syscalls.
+ */
 LIBCOM_API enum epicsSocketSystemCallInterruptMechanismQueryInfo 
         epicsSocketSystemCallInterruptMechanismQuery ();
 
 #ifdef EPICS_PRIVATE_API
-/*
- * Some systems (e.g Linux and Windows 10) allow to check the amount
- * of unsent data in the output queue.
- * Returns -1 if the information is not available.
+/*!
+ * \brief Query unsent data in a socket's output queue
+ *
+ * Query unsent data in a socket's output queue.  Some systems (e.g Linux and
+ * Windows 10) allow to check the amount of unsent data in the output queue.
+ *
+ * \param sock Socket to query for unsent data
+ * \return The number of bytes of unsent data in the output queue.  Returns -1 if the
+ * information is not available.
  */
 LIBCOM_API int epicsSocketUnsentCount(SOCKET sock);
 #endif
 
-/*
- * convert socket address to ASCII in this order
- * 1) look for matching host name and typically add trailing IP port
- * 2) failing that, convert to raw ascii address (typically this is a
+/*!
+ * \brief Convert socket address to ASCII
+ *
+ * Converts a socket address to an ASCII string.  Conversion occurs in this
+ * order:
+ * 1. look for matching host name and typically add trailing IP port
+ * 2. failing that, convert to raw ascii address (typically this is a
  *      dotted IP address with trailing port)
- * 3) failing that, writes "<Ukn Addr Type>" into pBuf
+ * 3. failing that, writes "<Ukn Addr Type>" into pBuf
  *
- * returns the number of character elements stored in buffer not
+ * \param paddr sockaddr structure to convert to ascii address:port
+ * \param[out] pBuf Pointer to the output buffer contantaining the address:port string
+ * \param bufSize Length of pBuf array
+ * \return The number of character elements stored in buffer not
  * including the null termination, but always writes at least a
  * null terminator in the string (if bufSize >= 1)
  */
 LIBCOM_API unsigned epicsStdCall sockAddrToA (
     const struct sockaddr * paddr, char * pBuf, unsigned bufSize );
 
-/*
- * convert IP address to ASCII in this order
+/*!
+ * \brief Convert IP address to ASCII
+ *
+ * Convert an IP address to an ASCII string.  Conversion occurs in this order:
  * 1) look for matching host name and add trailing port
  * 2) convert to raw dotted IP address with trailing port
  *
- * returns the number of character elements stored in buffer not
- * including the null termination, but always writes at least a
- * null terminator in the string (if bufSize >= 1)
+ * \param pInetAddr sockaddr_in structure to convert to address:port string
+ * \param[out] pBuf Pointer to the char array where the address:port string will be stored
```

Comment:
I think `:port` isn't correct for this case

Reply:
Thanks, nice catch.